apps/condo/domains/resident/schema/RegisterServiceConsumerService.js

```diff
@@ -60,12 +64,15 @@ const RegisterServiceConsumerService = new GQLCustomSchema('RegisterServiceConsu
                 //TODO(zuch): Ask about wrong logic - resident unit name do not match billing account unitName
                 const unitName = get(resident, ['unitName'])
 
+                const paymentCategory = get(extra, ['paymentCategory'], null)
```

Comment:
no need for array here in path

Reply:
Noted, thanks. Fixed in #b26dbc6f93370bf2c93afccbfceb77b0f4439e42